app/models/category.rb

```diff
@@ -13,4 +13,8 @@
 class Category < ActiveRecord::Base
   validates_length_of :description, :maximum => 2000
   validates_presence_of :userid, :name
+
+  def self.link
```

Comment:
this feels like a strange thing to put at the model layer..? but it's okay for now